paper/paper.md

```diff
@@ -0,0 +1,107 @@
+\autoref{fig2} shows a simple column model with an OCDR intervention (macroalgae growth) added after a warm-up period, which increases the carbon export of the system.
+
+![Here we show the results of a 1D model, forced by idealised light and mixing, which qualitatively reproduces the biogeochemical cycles in the North Atlantic.
+We then add kelp (500 frond / m² in the top 50 m of water) in December of the 2ⁿᵈ year (black vertical line) which causes an increase in air-sea carbon dioxide exchange and sinking export, as well as a change in the phytoplankton growth cycle.
+Figure made with `Makie.jl` [@makie]. \label{fig2}](column_example.png)
+
+The biologically active particles built into ``OceanBioME.jl`` are particularly useful for OCDR applications.
+Accurate carbon accounting is essential for assessing the effectiveness of OCDR strategies.
+Biologically active particles can be used to track carbon from a particular source while accounting for interactions with its surroundings.
+Biologically active particles can also be used to model OCDR deployment strategies including seaweed cultivation, alkalinity enhancement, and marine biomass regeneration.
+``OceanBioME.jl`` currently includes an extended version of the sugar kelp model presented by @broch:2012 as an example of the utility and implementation of these features.
+
+The implementation of OceanBioME.jl models allows for seamless integration with data assimilation packages, such as ``EnsembleKalmanProcesses.jl`` [@ekp]. This feature facilitates rapid calibration of model parameters, providing a powerful utility for integrating observations and models, with the potential to improve model skill and identify key sources of uncertainty.
+
+A key metric for the validity of biogeochemical systems is the conservation of elements such as carbon and nitrogen in the system.
+We therefore continuously test the implemented models in a variety of simple scenarios (i.e. isolated, with/without air-sea flux, with/without sediment) to ensure that conservation conditions are met, and we will continue to add tests for any new models.
+Additionally, we check ``OceanBioME.jl`` utilities through standard tests such as comparison to analytical solutions for light attenuation, and conservation of tracers for active particle exudation and sinking.
+
+Finally, this software is currently facilitating multiple research projects into ocean CDR which would have been significantly harder with other solutions.
+For example, Chen (In prep.) is using the active particle coupling provided to investigate the effects of location and planting density of kelp in the open ocean on their carbon drawdown effect, as in the example above.
+Additionally, Strong-Wright (In prep.) is using the coupling of both the biogeochemistry and easy interface to couple the physics to study flow interactions with a fully resolved giant kelp forest model including the effects on nutrient transport and distribution.
```

Comment:
```suggestion

The implementation of OceanBioME.jl models allows for seamless integration with data assimilation packages, such as ``EnsembleKalmanProcesses.jl`` [@ekp]. This feature facilitates rapid calibration of model parameters, providing a powerful utility for integrating observations and models, with the potential to improve model skill and identify key sources of uncertainty.

A key metric for the validity of biogeochemical systems is the conservation of elements such as carbon and nitrogen in the system.
We therefore continuously test the implemented models in a variety of simple scenarios (i.e. isolated, with/without air-sea flux, with/without sediment) to ensure that conservation conditions are met, and we will continue to add tests for any new models.
Additionally, we check ``OceanBioME.jl`` utilities through standard tests such as comparison to analytical solutions for light attenuation, and conservation of tracers for active particle exudation and sinking.

# Statement of need

``OceanBioME.jl`` was designed specifically to study ocean carbon dioxide removal (OCDR) strategies.
Assessing the effectiveness and impacts of OCDR is challenging due to the complexities of the interactions between the biological, chemical, and physical processes involved in the carbon cycle.
Moreover, field trials of OCDR interventions are generally small-scale and targeted, while the intervention required to have a climate-scale impact is regional or global.
We have built ``OceanBioME.jl`` to meet these challenges by creating tools that provide a modular interface to the different components within the ocean modelling framework provided by ``Oceananigans.jl``.
This allows easy access to a suite of biogeochemical models ranging from simple idealized to full-complexity models.
\autoref{fig2} shows a simple column model with an OCDR intervention (macroalgae growth) added after a warm-up period, which increases the carbon export of the system.

![Here we show the results of a 1D model, forced by idealised light and mixing, which qualitatively reproduces the biogeochemical cycles in the North Atlantic.
We then add kelp (500 frond / m² in the top 50 m of water) in December of the 2ⁿᵈ year (black vertical line) which causes an increase in air-sea carbon dioxide exchange and sinking export, as well as a change in the phytoplankton growth cycle.
Figure made with `Makie.jl` [@makie]. \label{fig2}](column_example.png)

The biologically active particles built into ``OceanBioME.jl`` are particularly useful for OCDR applications.
Accurate carbon accounting is essential for assessing the effectiveness of OCDR strategies.
Biologically active particles can be used to track carbon from a particular source while accounting for interactions with its surroundings.
Biologically active particles can also be used to model OCDR deployment strategies including seaweed cultivation, alkalinity enhancement, and marine biomass regeneration.
``OceanBioME.jl`` currently includes an extended version of the sugar kelp model presented by @broch:2012 as an example of the utility and implementation of these features.

Finally, this software is currently facilitating multiple research projects into ocean CDR which would have been significantly harder with other solutions.
For example, Chen (In prep.) is using the active particle coupling provided to investigate the effects of location and planting density of kelp in the open ocean on their carbon drawdown effect, as in the example above.
Additionally, Strong-Wright (In prep.) is using the coupling of both the biogeochemistry and easy interface to couple the physics to study flow interactions with a fully resolved giant kelp forest model including the effects on nutrient transport and distribution.
```
Perhaps this works?

Reply:
From JOSS docs

> A Statement of need section that clearly illustrates the research purpose of the software and places it in the context of related work.

The section above doesn't deal with the "and places it in the context of related work." Also, I'm not sure but you are suggesting to have the "Statement of need" section somewhere in the middle of the paper? I was thinking of the Statement of need section appearing straight after the Summary?

I think these three sentences

> ``OceanBioME.jl`` was designed specifically to study ocean carbon dioxide removal (OCDR) strategies. Assessing the effectiveness and impacts of OCDR is challenging due to the complexities of the interactions between the biological, chemical, and physical processes involved in the carbon cycle. Moreover, field trials of OCDR interventions are generally small-scale and targeted, while the intervention required to have a climate-scale impact is regional or global.

cover the "clearly illustrates the research purpose of the software" but don't really address the "places it in the context of related work".

Reply:
It's not very clear from the comment but I was thinking of putting it at the end? So we describe the code and what it can do, then in a new section do the statement of need.

It might be better if we try and write the statement of need, and then cut repetition out of the original?

Reply:
I think so. I think a statement of need at the top and the a paragraph at the end bridging why OceanBioME.jl achieves what the Statement of need was arguing for? That's what you mean? I like that.

Reply:
@jagoosw we haven't done this? is the JOSS waiting for this to move on with the review? 🤷‍♀️

Reply:
Oh yeah sorry I've been marking exams for the last two weeks so haven't done this yet

Reply:
I think JOSS is mainly waiting for reviewers to be found

Reply:
When we fix it we should post `@editorialbot generate pdf` in the JOSS pre-review repo so that they get a new version of the pdf, as they may be waiting for this as well.